CONTRIBUTING.md

```diff
@@ -1,31 +1,149 @@
 # How To Contribute
```

Comment:
[View this rendered in markdown](https://github.com/CrowdStrike/ember-toucan-core/blob/ec9b6c504e72c3195279121bc58e4764b42c76d7/CONTRIBUTING.md)